src/storage/StorageServer.cpp

```diff
@@ -48,10 +50,11 @@ std::unique_ptr<kvstore::KVStore> StorageServer::getStoreInstance() {
     options.cffBuilder_ = std::make_unique<StorageCompactionFilterFactoryBuilder>(schemaMan_.get(),
                                                                                   indexMan_.get());
     if (FLAGS_store_type == "nebula") {
-        auto nbStore = std::make_unique<kvstore::NebulaStore>(std::move(options),
-                                                              ioThreadPool_,
-                                                              localHost_,
-                                                              workers_);
+        auto nbStore = std::make_unique<kvstore::NebulaStore>(
+            std::move(options),
+            ioThreadPool_,
+            localHost_,
+            raftWorkers_ != nullptr ? raftWorkers_ : workers_);
```

Comment:
It is not enough.
Because the works here is only used by followers,  for leader,  it will reuse the works with outside.

A better way is for read requests,  use readPool_ inside StorageServiceHandler.

Reply:
make sense.